packages/lesswrong/components/common/HomeLatestPosts.tsx

```diff
@@ -152,7 +153,7 @@ const HomeLatestPosts = ({classes}: {classes: ClassesType}) => {
 
   const showCurated = isFriendlyUI || (isLW && reviewIsActive())
 
-  if (isLW && userIsAdmin(currentUser) && recombeeEnabledSetting.get()) {
+  if (isLW && currentUser && userHasRecombeeFrontpage(currentUser) && recombeeEnabledSetting.get()) {
```

Comment:
see https://github.com/ForumMagnum/ForumMagnum/pull/9030/files#r1544884349 (and in this case, we can even get rid of the `isLW` check, since that's included in `userHasRecombeeFrontpage`)